packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -107,7 +108,8 @@ def test_octarine_srv(cluster, timeout=30):
     cmd = ('/opt/mesosphere/bin/octarine {} & '.format(octarine_id) +
            '/opt/mesosphere/bin/python -m http.server ${PORT0}')
     raw_app_id = 'integration-test-app-octarine-srv-{}'.format(test_uuid)
-    check_command = ('curl --fail --proxy {} _{}._{}._tcp.marathon.mesos')
+    check_command = ('curl --fail --proxy {} ' +
```

Comment:
Our wrap width is 120 characters, this is under that / doesn't need to wrap

To match the rest of the codebase should probably be:
```python
    check_command = 'curl --fail --proxy {} _{}._{}._tcp.marathon.mesos.mydcos.directory'.format(
        proxy
        port_name
        raw_app_id)
```